### PR TITLE
Updates for the release of 23.0.0.5

### PR DESCRIPTION
**Pull request by wraschke**

### CODE REVIEW

File: releases/23.0.0.5/beta-instanton/Dockerfile.ubi.openjdk17 [hunk 1]
```diff
@@ -0,0 +1,101 @@
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-ea-jdk-ubi-amd64
```
Comment (tjwatson) [inline releases/23.0.0.5/beta-instanton/Dockerfile.ubi.openjdk17:1]: We must not have beta directories in the versioned release folders.

File: releases/latest/beta-instanton/Dockerfile.ubi.openjdk17 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-ea-jdk-ubi-amd64
 ARG LIBERTY_VERSION=23.0.0.5-beta
```
Comment (tjwatson) [inline releases/latest/beta-instanton/Dockerfile.ubi.openjdk17:2]: I expected the version to increase to 23.0.0.6-beta here.

File: releases/latest/beta/Dockerfile.ubi.openjdk17 [hunk 3]
```diff
@@ -1,7 +1,7 @@
 FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jdk-ubi
 ARG LIBERTY_VERSION=23.0.0.5-beta
```
Comment (tjwatson) [inline releases/latest/beta/Dockerfile.ubi.openjdk17:2]: I expected the version to increase to 23.0.0.6-beta here.

File: releases/latest/beta/Dockerfile.ubuntu.openjdk11 [hunk 4]
```diff
@@ -1,7 +1,7 @@
 FROM ibm-semeru-runtimes:open-11-jre-focal
 ARG LIBERTY_VERSION=23.0.0.5-beta
```
Comment (tjwatson) [inline releases/latest/beta/Dockerfile.ubuntu.openjdk11:2]: I expected the version to increase to 23.0.0.6-beta here.

File: releases/latest/beta/Dockerfile.ubuntu.openjdk17 [hunk 5]
```diff
@@ -1,7 +1,7 @@
 FROM ibm-semeru-runtimes:open-17-jre-focal
 ARG LIBERTY_VERSION=23.0.0.5-beta
```
Comment (tjwatson) [inline releases/latest/beta/Dockerfile.ubuntu.openjdk17:2]: I expected the version to increase to 23.0.0.6-beta here.

File: releases/23.0.0.5/beta-instanton/Dockerfile.ubi.openjdk17 (hunk 1 above)
Comment (wraschke) [reply releases/23.0.0.5/beta-instanton/Dockerfile.ubi.openjdk17:1]: I know. Your changes to InstantOn are so new and the automation script isn't updated to consider them.

**Pull request closed without merging**